libs/daohaus-connect-feature/src/HausConnectContext.tsx

```diff
@@ -47,7 +48,8 @@ export type UserConnectType = {
   networks: NetworkConfigs;
   switchNetwork: (chainId: string) => void;
   isProfileLoading: boolean;
-  isDaoScope: boolean;
+  setDaoChainId: React.Dispatch<SetStateAction<string | null>>;
```

Comment:
We've got a ReactSetter<> utility type to make this cleaner. Don't have to use it here, but for future reference.